controllers/v1/cloud-services/files.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * name : files.js
+ * author : prajwal
+ * created-date : 25-Apr-2024
+ * Description :  Files Controller.
+ */
+
+// dependencies
+let filesHelpers = require(MODULES_BASE_PATH + '/cloud-services/files/helper')
+const path = require('path')
+const fs = require('fs')
+/**
+ * Files service.
+ * @class
+ */
+
+module.exports = class Files {
+	/**
+	 * @apiDefine errorBody
+	 * @apiError {String} status 4XX,5XX
+	 * @apiError {String} message Error
+	 */
+
+	/**
+	 * @apiDefine successBody
+	 * @apiSuccess {String} status 200
+	 * @apiSuccess {String} result Data
+	 */
+
+	constructor() {}
+
+	static get name() {
+		return 'files'
+	}
+
+	/**
+     * @api {post} /project/v1/cloud-services/files/preSignedUrls  
+     * Get signed URL.
+     * @apiVersion 1.0.0
+     * @apiGroup Files
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiParamExample {json} Request:
+     * {
+     *  "request" : {
+     *  "5f72f9998925ec7c60f79a91": {
+     *     "files": ["uploadFile.jpg", "uploadFile2.jpg"]
+        }}
+      }
+     * @apiSampleRequest /project/v1/cloud-services/files/preSignedUrls
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * * {
```

Comment:
@Prajwal17Tunerlabs response is correct right?

Reply:
yes, i checked the response object, its correct